README.md

```diff
@@ -1,12 +1,16 @@
-# Git Repository Template
+# Github Notifications Application
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+## What is Github Notifications Application?
+Github Notifications is a macOS application to manage Github Notifications to be more personalized. Receive notifications on your Mac when there are new updates on subscribed repositories.
```

Comment:
```suggestion
Github Notification Application is a macOS application for managing and personalized Github's notifications.
```

Reply:
In the second sentence, I suggest you list down the actions that trigger to show notification, f.e: mention, request review, etc.